src/connector/postgres/conversion.rs

```diff
@@ -477,7 +477,10 @@ impl<'a> ToSql for Value<'a> {
             (Value::Array(decimals), &PostgresType::FLOAT8_ARRAY) => decimals.as_ref().map(|decimals| {
                 let f: Vec<f64> = decimals
                     .into_iter()
-                    .filter_map(|v| v.as_decimal().and_then(|decimal| decimal.to_f64()))
+                    .filter_map(|v| {
+                        v.as_decimal()
+                            .and_then(|decimal| decimal.to_string().parse::<f64>().ok())
+                    })
```

Comment:
In this case, it looks like we are doing &[f64] -> &[Decimal] -> &[String] -> Vec<f64> (it's not the case, but schematically). Would it be possible to not convert to decimal instead? v.as_f64() directly, maybe?

Reply:
Wait, no, this goes from `Value::Real(Decimal)` -> `Decimal` -> `String` -> `f64`...

Reply:
ah indeed, if it's already a decimal we have to do that